vcd/resource_vcd_vapp_vm.go

```diff
@@ -465,6 +473,49 @@ func vmSchemaFunc(vmType typeOfVm) map[string]*schema.Schema {
 				},
 			}},
 		},
+		"boot_options": {
```

Comment:
`vcd.ResourceSchema-vcd_vapp_vm.tf` and `vcd.ResourceSchema-vcd_vm.tf` binary tests will have to be added to `skip-upgrade-tests.txt`

Reply:
Added.